gulpfile.js

```diff
@@ -0,0 +1,68 @@
+const gulp = require('gulp');
+
+const browserSync = require('browser-sync').create();
+
+const testSync = require('browser-sync').create();
+
+const reload = browserSync.reload;
+
+const browserify = require('gulp-browserify');
+
+const rename = require('gulp-rename');
+
+const path = require('path');
+
+const karma = require('karma').Server;
+
```

Comment:
Nitpick: remove the empty line between each of your constant assignments; not necessary.